styles.css

```diff
@@ -456,7 +456,7 @@ li {
 }
 
 .about-me-title {
```

Comment:
[OPTIONAL] I think it would be better if this blank space in the bottom part of your about me section gets removed:

![extraspace](https://user-images.githubusercontent.com/111214189/229314564-b7012ae2-b145-497f-9ab7-69604026fdc3.png)

This will make your project look more professional and also follow the figma design and prepare your project for the next milestone 👍